### PR TITLE
[RFR] Added “DOM” entry

### PR DESCRIPTION
**Pull request by jalbertbowden**
Document Object Model definition

### CODE REVIEW

**Comment by KittyGiraudel** [comment]
Do you mind adding a link to the README maybe? :)

File: glossary/DOM.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[DOM (Document Object Model)](http://www.w3.org/DOM/) is a platform- and language-neutral interface that will allow programs and scripts to dynamically access and update the content, structure and style of documents. The document can be further processed and the results of that processing can be incorporated back into the presented page. This is an overview of DOM-related materials here at W3C and around the web.
```
Comment (KittyGiraudel) [inline glossary/DOM.md:1]: Miss `# DOM` as a title on first line, then an empty line on line 2. :)

**Comment by KittyGiraudel** [comment]
All good, thank you! :)

**Comment by jalbertbowden** [comment]
sure thing...where?  
sorry for asking, but i certainly can do it, and i'm guessing should have done it, i just don't see it in the AJAX or Angular defintions....  
more than happy to comply

**Comment by KittyGiraudel** [comment]
I did it myself, no problem. :)

**Pull request merged**